src/components/HelloWorld.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <div id="container">
+      <h1>Menu of Salads</h1>
+      <ul>
+        <li v-for="(item, index, price) in myJson" :key="index" :id="index">
+          {{ index + 1 }}. {{ item }}
+          <div class="lisupport">
+            <p v-if="index % 3 == 0">{{ (price = +15) }}</p>
+            <p v-else-if="item.startsWith('E')">{{ (price = +10) }}</p>
+            <p v-else-if="item.startsWith('R')">{{ (price = +10) }}</p>
+            <p v-else-if="item.startsWith('G')">{{ (price = +10) }}</p>
+            <p v-else>{{ (price = +20) }}</p>
+            <input
+              type="checkbox"
+              :id="index"
+              :value="price"
+              :checked="false"
+              @click="check($event)"
+              number
+            />
+          </div>
+        </li>
+        <!-- <p>Total: {{  }}</p> -->
+      </ul>
+      <p class="total">Total: {{ total }}</p>
+    </div>
+    <footer>Copyright &copy; Rafal Weglinski 2020 RW&reg;</footer>
+  </div>
+</template>
+
+<script>
+import json from "./../../json/salads.json";
```

Comment:
Tak. To jest jak najbardziej valid, ale chciałem, żebyście potraktowali salads jak zasób i spróbowali go ściągnąć. Idealna okazja do przetestowania axiosa :D !

Reply:
Druga sprawa to nazwa. Json mówi nam tylko o formacie. Czemu nie nazwać tego tym czym to jest. Np. salads, albo saladList. Od razu jaśniej.